src/store/metadata/stub/content/editions/versions/annual-personal-wellbeing-estimates_2023-09 .json

```diff
@@ -0,0 +1,88 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates/editions/2023-09/versions",
+    "@type": "hydra:Collection",
+    "title": "Annual personal wellbeing estimates",
+    "versions": [
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates/editions/2023-09/versions/1",
+            "identifier": "1",
+            "issued": "2023-09-13T13:32:00Z",
+            "title": "Annual personal wellbeing estimates",
+            "summary":  "Estimates of life satisfaction, feeling that the things done in life are worthwhile, happiness and anxiety for adults aged 16 years and over, UK, 04/2011-03/2012 to 04/2022-03/2023.",
+            "description": "This dataset shows the average rating of those that feel satisfied about their lives (where 0 is 'not at all satisfied' and 10 is 'completely satisfied'), for adults aged 16 years and over, in the UK, for periods 04/2011-03/2012 to 04/2022-03/2023.",
+            "download_url": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates/editions/2023-09/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+              "columns": [
+                {
+                  "name": "period",
+                  "datatype": "datetime",
```

Comment:
The period column uses the datetime datatype, but the CSV does not use time.

Reply:
done